features/121-Portal/Change_password.feature

```diff
@@ -0,0 +1,34 @@
+@ho-portal
+
+Feature: Change password
+
+  Background:
+    Given a logged-in user
+    And "Change password" page is open
+
+  Scenario: Change password succesfully
+    Given user has entered current password correctly
+    Given Existing password field apperas green when active
+    When user enters new password
+    And new password is at least 8 characters one capital letter, one lowercase letter and one number
+    And new password field turns green when avtive
+    And user enters same password in "Confirm password" field
+    And confirm password field turns green when avtive
+    And in both fields password match
+    And update button turns black
+    Then when user clicks on "Update" button password is changed
+    And user is navigated to different page
+    And "Password changed" is displayed
+
+  Scenario: Change password unsuccesfully
+    When user has entered current password incorrectly
+    And existing password is not at least 8 characters one capital letter, one lowercase letter and one number
```

Comment:
I don't think it's very insightful to repeat all the exact same things as above here. I get that you need to fill in the new password correctly to be able to hit 'Update', but still it's exactly the same as above, and it's just noise then. I would simply refer to the above scenario with something like 'Given for the rest everything the same as in previous scenario'